config/database.yml

```diff
@@ -17,6 +17,11 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+<<<<<<< HEAD
+  host: <%= ENV['DATABASE_HOST'] %>
```

Comment:
The `database.yml` does not need to be in the PR because the master have the file updated to work in development environment without any additional configuration.